packages/core-app-api/src/routing/RouteResolver.ts

```diff
@@ -180,14 +180,14 @@ function resolveBasePath(
 
 export class RouteResolver {
   constructor(
-    private readonly routePaths: Map<RouteRef, string>,
-    private readonly routeParents: Map<RouteRef, RouteRef | undefined>,
-    private readonly routeObjects: BackstageRouteObject[],
-    private readonly routeBindings: Map<
+    public readonly routePaths: Map<RouteRef, string>,
+    public readonly routeParents: Map<RouteRef, RouteRef | undefined>,
+    public readonly routeObjects: BackstageRouteObject[],
+    public readonly routeBindings: Map<
       ExternalRouteRef,
       RouteRef | SubRouteRef
     >,
-    private readonly appBasePath: string, // base path without a trailing slash
+    public readonly appBasePath: string, // base path without a trailing slash
```

Comment:
These things are very much internal btw, let's not make them public in this form, but see my other comment

Reply:
Right, yeah I understand as little exposing of internals as possible makes sense, I do use `routeObjects` and `routeBindings` in the [DevTools plugin](https://github.com/backstage/backstage/pull/10260) though (to show routes and route info), but if the same information can be extracted in a more "public" fashion, that'd certainly be ok too.